src/main/app/case/case.ts

```diff
@@ -136,10 +136,27 @@ export interface Case {
   childrenSexAtBirth?: Gender;
   childrenNationality?: string[];
   childrenAdditionalNationalities?: string[];
-
   childrenFirstNameAfterAdoption?: string;
   childrenLastNameAfterAdoption?: string;
 
+  birthFathersNameOnCertificate?: string;
+  birthFatherFirstNames?: string;
+  birthFatherLastNames?: string;
+  birthFatherStillAlive?: string;
+  birthFatherUnsureAliveReason?: string;
```

Comment:
please rename birthFatherUnsureAliveReason to  birthFatheNotAliveReason similar to birthMother done by Abid (Please check his PR once)

Reply:
This naming convention would not make sense. The text field is for when the applicant selects 'Not sure' rather than 'No' in answering whether the birth father is alive. I suggest reviewing with the team whether birthMother naming should be changed